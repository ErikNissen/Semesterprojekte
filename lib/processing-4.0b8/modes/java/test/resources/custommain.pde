version https://git-lfs.github.com/spec/v1
oid sha256:450237ca0834327e39fbce1b3e99d8e0d4e398851de5263333af63e04fbf3225
size 76
