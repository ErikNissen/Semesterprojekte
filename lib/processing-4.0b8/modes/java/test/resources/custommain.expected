version https://git-lfs.github.com/spec/v1
oid sha256:72b3c0ac6c73edaf88ce7986809c04f70d1f1425ab8632b3d3eb0df1b6606b4d
size 451
